modules/grafana/locals.tf

```diff
@@ -36,4 +36,64 @@ locals {
     },
   ]
 
+
+  default_datasource_configs = {
+    prometheus = {
+      type        = "prometheus"
+      name        = "Prometheus"
+      access_mode = "proxy"
+      url         = "http://prometheus-operated.monitoring.svc.cluster.local:9090"
+      is_deafult  = false
+    }
+    cloudwatch = {
+      type        = "cloudwatch"
+      name        = "Cloudwatch"
+      access_mode = "proxy"
+      jsonData = {
+        authType      = "default"
+        defaultRegion = "us-east-2"
+        assumeRoleArn = null
+      }
+      is_default = false
+    }
+    loki = {
+      type        = "loki"
+      name        = "Loki"
+      access_mode = "proxy"
+      url         = "http://loki.monitoring.svc.cluster.local:3100"
```

Comment:
lets have the namespace here in endpoints based on release namespace instead of hardcoded `monitoring`